Software.py

```diff
@@ -436,29 +436,10 @@ def initializePlugin(initializedAnonUser, serverAvailable):
 
     oneMin = 60
 
-    setInterval(sendOfflineData, oneMin * 15)
     setInterval(lambda: sendHeartbeat('HOURLY'), oneMin * 60)
-    setInterval(sendOfflineEvents, oneMin * 40)
-    setInterval(getUsersOfFirstProject, oneMin * 50)
-
-    # only send commit data if setting is enabled
-    disable_git_data = getItem("disableGitData")
```

Comment:
Let's make sure we use this setting when we reimplement Git events

Reply:
ok, sounds good